spot_rl_experiments/spot_rl/models/owlvit.py

```diff
@@ -11,24 +11,64 @@
 import cv2
 import torch
 from PIL import Image
-from transformers import OwlViTForObjectDetection, OwlViTProcessor
+from transformers import (
+    Owlv2ForObjectDetection,
+    Owlv2Processor,
+    OwlViTForObjectDetection,
+    OwlViTProcessor,
+)
 
 
 class OwlVit:
-    def __init__(self, labels, score_threshold, show_img):
+    def __init__(
+        self,
+        labels,
+        score_threshold,
+        show_img,
+        version: int = 1,
+    ):
+        if version < 1 or version > 2:
+            raise ValueError(f"OWL-ViT version can only be 1 or 2. Received: {version}")
+        self._version = version
         # self.device = torch.device('cpu')
         self.device = (
             torch.device("cuda") if torch.cuda.is_available() else torch.device("cpu")
         )
+        if version == 1:
```

Comment:
Is the version one be the one we used before?

Reply:
Yes, we were using Owlvit i.e. version1 before. Aria code uses OwlV2 or version2 but this may change again in near future.